test/scripts/H2CO/compare_results.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+folder1=$1
+folder2=$2
+
+process_quantum_energies () {
+  filename=$1
+
+  fingerprints=$(awk '/^Start Fingerprints/,/^End Fingerprints/ {print $0}' $filename)
+  printf "$fingerprints"
+  echo -e "\n"
+
+  quantum_numbers=$(awk '/^Start Quantum/,/^End Quantum/ {print $0}' $filename)
+  printf "$quantum_numbers" | awk '!/number|Class/ {printf("%0.6e\n", $5);}'
+}
+
+process_simple_files () {
```

Comment:
Having a brief description of what we're comparing here would be helpful for future maintenance.